### DISCUSSION TITLE
Make a job search app

### DISCUSSION THREAD

**Discussion by thienvc** [2sic/2sxc#2630] [General]
I want to make a job search app, users can post their CV, and businesses can post recruitment information, is there an app that can be quickly modified?

**Comment by iJungleboy** [comment]
Hi @thienvc - ATM there is no quick-start for this complete scenario. 

Basically you need a system where people can create their own data and possibly edit it as well. 

To create data I suggest you start with the Mobius App. If you modify it, so that only authenticated users can create data, then as you'll see the created data can be 'owned' by the person creating it. You can then decide if further editing can be done in the normal UI (using the aforementioned permissions) or through custom forms you create as you wish - then saving through a custom web-api as you see fit. 

Searching etc. is then fairly simple, as you can do whatever you want.

**Reply by thienvc** [reply]
Thanks for your reply
I'm reviewing the Events And Courses V6 .1 app, I don't understand why I set the default and I can't see the Event details.
Event Detail link `/details/2881/datitem-eid2889/demo-entry` has been redirect to `/datitem-eid2889/demo-entry`, and `/details/2881` was automatically removed from the URL for unknown reasons
